src/perps/positions.ts

```diff
@@ -1,6 +1,5 @@
-import Decimal from 'decimal.js';
-import { Position } from '../interfaces/sdkTypes';
-import { formatEther } from 'ethers';
+import Decimal from "decimal.js";
+import { Position } from "../interfaces/sdkTypes";
```

Comment:
please can we simplified thus function. 

1. We dont need the complete position to make this calculation. Ask for the minimum information needed
2. We can ask for the params on number instead of Decimal. We can convert to decimal inside the function

similar to ```calculatePositionLeverage```